content/en/docs/concepts/containers/images.md

```diff
@@ -157,6 +157,44 @@ that Kubernetes will keep trying to pull the image, with an increasing back-off
 Kubernetes raises the delay between each attempt until it reaches a compiled-in limit,
 which is 300 seconds (5 minutes).
 
+## Serial and parallel image pulls
+
+By default, kubelet pulls images serially. In other words, kubelet will send only
+one image pull request to the image service at a time. Other image pull requests
+have to wait until the one being processed is complete.
+
+Nodes make image pull decisions in isolation. Even when you use serialized image pulls,
+two different nodes can pull the same image in parallel.
```

Comment:
In my other comment I wanted to highlight that when Pod has Init contianer and regular container, image download for those will not be parallelized. Those will be sequencial. Can you please add this note?